nmostesting/suites/IS1101Test.py

```diff
@@ -0,0 +1,3813 @@
+            try:
+                effective_edid_before = self.get_effective_edid(test, inputId)
+
+                valid, response = self.do_request("PUT",
+                                                  self.compat_url + "inputs/" + inputId + "/edid/base",
+                                                  headers={"Content-Type": "application/octet-stream"},
+                                                  data=self.valid_edid,
+                                                  params={"adjust_to_caps": "true"})
+                if not valid or response.status_code != 204:
+                    return test.FAIL("Unexpected response from "
+                                     "the Stream Compatibility Management API: {}".format(response))
+
+                result = self.wait_until_true(
+                    partial(is_edid_inequal_to_effective_edid, self, test, inputId, effective_edid_before)
+                )
+                if not result:
+                    return test.FAIL("Effective EDID doesn't change when Base EDID changes")
+
+                valid, response = self.do_request("DELETE", self.compat_url + "inputs/" + inputId + "/edid/base")
+                if not valid or response.status_code != 204:
+                    return test.FAIL("Unexpected response from "
+                                     "the Stream Compatibility Management API: {}".format(response))
+
+                result = self.wait_until_true(
+                    partial(is_edid_equal_to_effective_edid, self, test, inputId, effective_edid_before)
+                )
+                if not result:
+                    return test.FAIL("Effective EDID doesn't restore after Base EDID DELETion")
+
+            except json.JSONDecodeError:
+                return test.FAIL("Non-JSON response returned from Node API")
+            except KeyError as e:
+                return test.FAIL("Unable to find expected key: {}".format(e))
+        return test.PASS()
+
+    # SENDERS TESTS
+    """
+    Runs Node Tests covering IS-11 for Senders
+    """
+
+    def get_another_grain_rate(self, grain_rate):
+        numerator = grain_rate["numerator"]
+        denominator = grain_rate["denominator"]
+        if (numerator == 30 or numerator == 25) and denominator == 1:
+            return {"numerator": numerator * 2, "denominator": 1}
+        if (numerator == 60 or numerator == 50) and denominator == 1:
+            return {"numerator": numerator / 2, "denominator": 1}
+        if numerator == 24 and denominator == 1:
+            return {"numerator": 30, "denominator": 1}
+        if (numerator == 30000 or numerator == 25000) and denominator == 1001:
+            return {"numerator": numerator * 2, "denominator": 1001}
+        if (numerator == 60000 or numerator == 50000) and denominator == 1001:
+            return {"numerator": numerator / 2, "denominator": 1001}
+        return "grain_rate not valid"
```

Comment:
`get_another_grain_rate` assumes that the Node under test supports all these grain rates. 
Similarly `get_another_sample_rate` assumes the Node supports all the defined sample rates.

Reply:
If the node does not support the grain rate is will refuse the constraint and the test should be skipped (once the implementer fix the previously observed bug when testing the return status :). Note that there is yet no Sender Capabilities in NMOS.

Reply:
Our experience is that the test aren't being skipped ('Could Not Test') but being failed when the Node doesn't support a particular grain rate...